cmake/configs/default.cmake

```diff
@@ -58,6 +58,7 @@ hunter_config(CppNetlib VERSION 0.10.1-hunter-3)
 hunter_config(CppNetlibUri VERSION 1.0.4-hunter)
 hunter_config(CsvParserCPlusPlus VERSION 1.0.1)
 hunter_config(Eigen VERSION 3.3.4-p0)
+hunter_config(state_machine VERSION 1.1)
```

Comment:
Hi, could we have alphabetical order, following the ASCII order, in default.cmake? (in doubt: http://www.asciitable.com/ )

There are some packages out of ascii order, although in alphabetical order, which already makes things incongruent, but placing libs completely random in the file like this hinders maintenance .

Thanks

Reply:
@pretyman it makes sense. Next time I will fix it, thanks